TerriaJS/terriajs
# Difftool fixes

### What this PR does

- Uses `null` instead of `undefined` to unset `currentTime` from DiffTool.
- Fixes `CzmlCatalogItem` to react to input data changes. Previously this resulted in marker location not updating on clicking a different location when using difftool.


### Testing

- Open this share [link](http://ci.terria.io/difftool-fixes/#clean&share=s-z1VBtkLJqLxdRg0Mml6jYI73Jtu&https://gist.githubusercontent.com/na9da/131cb6308113ec3fc3e4a1211e2ecca3/raw/1c6e8f5c29db3afb970978a3cde3cedfc0577c31/sentinel.json).
- Open the "Difference" tool from the context menu for the first item.
- Try to generate a difference image
- Everything should work.


This is a description of what I've done in this PR, especially explaining choices made to make the reviewer's job easier. If I don't replace this paragraph with an actual description, my PR will probably be ignored.

### Checklist
